platform/android/gradle/dependencies.gradle

```diff
@@ -14,7 +14,7 @@ ext {
             leakCanary      : '1.6.1',
             junit           : '4.12',
             mockito         : '2.23.4',
-            mockk           : '1.8.13.kotlin13',
+            mockk           : '1.9.kotlin12',
```

Comment:
we are using kotlin1.3 in https://github.com/mapbox/mapbox-gl-native/blob/6b5b3508aa7afc1ea1710baf6af87526d8c4803e/platform/android/gradle/dependencies.gradle#L21

So this dependency should be 1.9 instead:

![image](https://user-images.githubusercontent.com/2151639/51982263-bf511580-2495-11e9-97cb-9aade6255bdf.png)

Reply:
whoops :+1: